hypothesis-python/src/hypothesis/internal/conjecture/shrinker.py

```diff
@@ -528,12 +534,13 @@ def s(n):
 
                         self.debug(
                             "  * %s made %d call%s of which "
-                            "%d shrank, deleting %d byte%s."
+                            "%d shrank and %d were misaligned, deleting %d byte%s."
```

Comment:
As above.  We might also use f-strings here (which tells you how long it's been since anyone touched this...)